packages/forma-36-react-components/src/components/Modal/ModalHeader/ModalHeader.css

```diff
@@ -8,17 +9,16 @@
   flex-shrink: 0;
   align-items: center;
   /* prettier-ignore */
```

Comment:
Not related, but this might be not needed anymore?

Reply:
removed